### PR TITLE
Expand on applicability of this scheme for clarity

### PR DESCRIPTION
**Pull request by LPardue**
Addressed E#3a﻿ of #1802

Reference the HTTP terms we are using. Be clearer that we focus on "normal" responses and that there are other possible uses. But avoid making the entire document a piece of abstract art.

### CODE REVIEW

File: draft-ietf-httpbis-priority.md [hunk 1]
```diff
@@ -248,12 +248,24 @@ behind the server that the client is directly connected to.
...
+The CONNECT method (see {{Section 9.3.6 of HTTP}}) can be used to establish
+tunnels. Signaling applies similarly to tunnels; additional considerations for
+server prioritization are given in {{connect-scheduling}}.
+
+{{client-scheduling}} describes how clients can optionally apply elements of
+this scheme locally to the request messages that they generate.
+
+Some forms of HTTP extensions change HTTP/2 or HTTP/3 stream behavior or
+define new data carriage mechanisms. Such extensions are not considered by this
+document; they can define themselves how this priority scheme might be applied.
```
Comment (kazuho) [inline draft-ietf-httpbis-priority.md:268]:
```suggestion
Some forms of HTTP extensions might change HTTP/2 or HTTP/3 stream behavior or
define new data carriage mechanisms. Such extensions can define themselves how
this priority scheme is to be applied.
```

I would assume that this is hand-waving to extensions that are _yet_ to be standardized. Then, we "cannot" consider how this prioritization scheme will be applied to them.

**Review by kazuho — COMMENTED** [review]
Thank you for the PR. This looks good.

File: draft-ietf-httpbis-priority.md [hunk 2]
```diff
@@ -248,12 +248,24 @@ behind the server that the client is directly connected to.
...
 
-The priority scheme defined by this document considers only the prioritization
-of HTTP messages and tunnels, see {{client-scheduling}}, {{server-scheduling}},
-and {{connect-scheduling}}.
-
-Where HTTP extensions change stream behavior or define new data carriage
-mechanisms, they can also define how this priority scheme can be applied.
+The priority scheme defined by this document is primarily focused on the
+prioritization of HTTP response messages (see {{Section 3.4 of HTTP}}). It
+defines new priority parameters ({{parameters}}) and signals ({{header-field}}
```
[the remark this answers is not in the document]
Comment (LPardue) [reply draft-ietf-httpbis-priority.md:253]: WFM

File: draft-ietf-httpbis-priority.md (hunk 1 above)
Comment (LPardue) [reply draft-ietf-httpbis-priority.md:268]: much neater, thanks!

**Review by kazuho — APPROVED** [review]

**Pull request merged**